packages/@aws-cdk/assert/lib/synth-utils.ts

```diff
@@ -7,7 +9,9 @@ export class SynthUtils {
   }
 
   public static templateForStackName(session: ISynthesisSession, stackName: string): any {
-    return session.store.readJson(session.getArtifact(stackName).properties!.templateFile);
+    const fileName = session.getArtifact(stackName).properties!.templateFile;
```

Comment:
I worry about this `!` here. Can we *guarantee* that `properties` will *always* be present? It also seems odd that a `!` isn't needed for `session.getArtifact(stackName)` - can there somehow not be such a thing? Is it guaranteed to always be a valid thing?

Reply:
Cleaned this up